daisy_workflows/image_build/windows/windows-81-ent-x64-uefi.wf.json

```diff
@@ -0,0 +1,113 @@
+{
+  "Name": "windows-windows-81-ent-x64-uefi",
+  "Vars": {
+    "install_disk": "install-disk",
+    "media": {
+      "Required": true,
+      "Description": "GCS or local path to Windows Server ISO."
+    },
+    "pwsh": {
+      "Required": true,
+      "Description": "GCS or local path to PowerShell v7+ installer"
+    },
+    "dotnet48": {
+      "Required": true,
+      "Description": "GCS or local path to Microsoft .NET Framework 4.8 offline installer"
+    },
+    "cloudsdk": {
+      "Required": true,
+      "Description": "GCS or local path to Cloud SDK installer"
+    },
+    "description": {
+      "Value": "Microsoft, Windows 8.1 Enterprise, x64 built on ${TIMESTAMP}, supports Shielded VM features"
+    },
+    "family": {
+      "Value": "windows-81",
+      "Description": "Desired image family of the output image."
+    },
+    "google_cloud_repo": {
+      "Value": "stable",
+      "Description": "Google Cloud repo to retrieve packages from during the build."
+    },
+    "name": {
+      "Value": "windows-81-ent-x64-uefi-v${TIMESTAMP}",
+      "Description": "The name of the output image."
+    },
+    "project": {
+      "Value": "${PROJECT}",
+      "Description": "The GCP project to create and store the image in."
+    },
+    "updates": {
+      "Value": "",
+      "Description": "GCS or local filesystem location containing Windows update files."
+    },
+    "drivers_bucket": {
+      "Value": "gs://gce-windows-drivers-public/release/win6.3-signed-stornvme/",
+      "Description": "GCS location containing the GCP Windows driver files."
+    },
+    "install_disk_size": {
+      "Value": "50",
+      "Description": "The size of disk to provision for the image in GB."
+    },
+    "workflow_root": {
+      "Value": "/workflows",
+      "Description": "Root of github workflows, defaults to /workflows in the container."
+    }
+  },
+  "Steps": {
+    "windows-build": {
+      "Timeout": "5h",
+      "IncludeWorkflow": {
+        "Path": "./windows-build-uefi.wf.json",
+        "Vars": {
+          "install_disk": "${install_disk}",
+          "install_disk_size": "${install_disk_size}",
+          "updates": "${updates}",
+          "drivers_bucket": "${drivers_bucket}",
+          "dotnet48": "${dotnet48}",
+          "pwsh": "${pwsh}",
+          "edition": "Windows 8.1 Enterprise",
+          "media": "${media}",
+          "cloud_sdk": "${cloudsdk}",
+          "google_cloud_repo": "${google_cloud_repo}",
+          "workflow_root": "${workflow_root}"
+        }
+      }
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Project": "${project}",
+          "SourceDisk": "${install_disk}",
+          "Name": "${name}",
+          "Family": "${family}",
+          "Description": "${description}",
+          "Licenses": [
+            "projects/windows-cloud/global/licenses/windows-81-x64-byol"
+          ],
+          "GuestOsFeatures": [
```

Comment:
You have GVNIC in the other review as well

Reply:
We use GVNIC in the publish builds that we run, but I believe it is omitted from these because these are templates external users can copy/paste to a much larger extent. I'm not sure we want to default them all into GVNIC just yet (unless we do)?

Reply:
sgtm